sgkit/stats/association.py

```diff
@@ -0,0 +1,160 @@
+import collections
+from typing import Optional, Sequence
+
+import dask.array as da
+import numpy as np
+import xarray as xr
+from dask.array import Array, stats
+from xarray import Dataset
+
+from ..typing import ArrayLike
+
+LinearRegressionResult = collections.namedtuple(
```

Comment:
Would it be better to use `dataclass` (more type safe)?

Reply:
https://github.com/pystatgen/sgkit/pull/16/commits/273375e5d949ae3cd6b22572a6fccb2bd3064f0f